src/buildercore/s3.py

```diff
@@ -77,5 +78,10 @@ def download(key, output_path):
     assert not os.path.exists(output_path), "given output path exists, will not overwrite: %r" % output_path
     k = builder_bucket().get_key(key)
     assert k, ("Cannot find %s in bucket %s" % (key, builder_bucket()))
+    _log_for_key(key, "downloading key %s")
     k.get_contents_to_file(open(output_path, 'w'))
     return output_path
+
+def _log_for_key(key, message):
```

Comment:
I don't think we need this function. It doesn't improve readability and we don't gain much.